cdc/owner/changefeed.go

```diff
@@ -125,9 +125,14 @@ type changefeed struct {
 	newScheduler func(ctx cdcContext.Context, startTs uint64) (scheduler.Scheduler, error)
 }
 
-func newChangefeed(id model.ChangeFeedID, up *upstream.Upstream) *changefeed {
+func newChangefeed(
+	id model.ChangeFeedID,
+	state *orchestrator.ChangefeedReactorState,
```

Comment:
please add a comment in changefeed struct to tell `state` is a read-only fileld.